metricbeat/module/mysql/_meta/supported-versions.yml

```diff
@@ -0,0 +1,8 @@
+variants:
+  - {'MYSQL_VARIANT': 'mysql',   'MYSQL_VERSION': '5.7.12'}
```

Comment:
To be consistent across modules, I'd use this format:
```yaml
variants:
  - {'IMAGE': 'mysql',   'VERSION': '5.7.12'}
  - {'IMAGE': 'mysql',   'VERSION': '8.0.13'}
```
This way the parser is abstract

Reply:
Then we'll have to change all build args used at the moment, does it worth it?

To what parser do you refer?

Reply:
Mmmm... you are right. The parser must use `MYSQL_VARIANT` (key of the map) as build-arg.

Thanks!